src/walletSdk/Anchor/index.ts

```diff
@@ -239,17 +247,29 @@ export class Anchor {
    * @throws [ServerRequestFailedError] if server request fails
    */
 
-  async getHistory(
-    params: GetTransactionsParams
-  ): Promise<AnchorTransaction[]> {
-    const { assetCode } = params;
-
+  async getHistory({
+    authToken,
+    assetCode,
+    noOlderThan,
```

Comment:
This arguments below are optional right?

Reply:
Yes! You can check it [here](https://github.com/stellar/typescript-wallet-sdk/blob/cg-types/src/walletSdk/Types/anchor.ts#L92):

```
export type GetTransactionsParams = {
  authToken: string;
  assetCode: string;
  noOlderThan?: string;
  limit?: number;
  kind?: string;
  pagingId?: string;
  lang?: string;
};
```